packages/flame_spine/lib/src/spine_component.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:ui';
+
+import 'package:flame/components.dart';
+import 'package:flame_spine/flame_spine.dart';
+import 'package:flutter/foundation.dart';
+
+class SpineComponent extends PositionComponent {
+  final SkeletonRender renderer;
+
+  SpineComponent({
+    required this.renderer,
+    required Vector2 super.size,
+    super.position,
+  });
+
+  @override
+  @mustCallSuper
+  void render(Canvas canvas) {
+    renderer.render(canvas, size.toSize());
```

Comment:
This should not do `toSize` on every render tick, it will create lots of objects, add a listener to `size` and update a private variable with the new size once it is updated and then use that private variable here.